### DISCUSSION TITLE
PhotoSauce.WebRSize only support net framework?

### DISCUSSION THREAD

**Discussion by AdwardNguyen** [saucecontrol/PhotoSauce#68] [Q&A] [answered]

**Comment by AdwardNguyen** [comment]
I try PhotoSauce.WebRSize in my project asp.net core 3 but it not available in dependencies because only support net framework version!

**Comment by saucecontrol** [comment, marked as answer]
That is correct, as mentioned in the note at the top of the docs: https://docs.photosauce.net/web.html

WebRSize is tightly integrated with IIS for performance reasons, and the APIs used for that integration do not exist in .NET Core.  Since .NET Core 2.2, it is possible to host ASP.NET Core inside an IIS worker process, but this would need to be built from scratch.  I will do that eventually, but it's not high on my priority list.

If you are hosting ASP.NET Core with Kestrel, the approach would be different, and it is relatively simple to build a functional middleware component to map URLs to MagicScaler settings.
